compileall2.py

```diff
@@ -384,6 +400,9 @@ def main():
                               'Python interpreter itself (specified by -O).'))
     parser.add_argument('-e', metavar='DIR', dest='limit_sl_dest',
                         help='Ignore symlinks pointing outsite of the DIR')
+    parser.add_argument('-hd', '--hardlink-dupes', action='store_true',
```

Comment:
Doubleletter short switch? Is that customary? It feels kinda weird.

Reply:
Just an option. We can remove it.